docs/content/operating.html.md

```diff
@@ -148,11 +148,16 @@ route, add the `--routes` flag.
 ## Monitoring
 
 Flynn provides a status endpoint over HTTP that exposes the health of system
-components at `http://status.$CLUSTER_DOMAIN` (for example,
-`http://status.1.localflynn.com`). The status endpoint returns a status code
+components at `http://status.$CLUSTER_DOMAIN?key=$auth_key` (for example,
+`http://status.1.localflynn.com?key=$auth_key`). The status endpoint returns a status code
```

Comment:
This is only necessary if the request doesn't come from a private IP, which should be noted here. Also, the convention we are using is all caps variables, so `$AUTH_KEY`.